NAS-101229: Add text about Cron job command length limit. (#1442)
freenas/freenas-docs: userguide/tasks.rst

   |                     |                             |                                                                                                         |
   +---------------------+-----------------------------+---------------------------------------------------------------------------------------------------------+
   | Command             | drop-down menu              | Enter the **full path** to the command or script to be run. If it is a script, testing it at the        |

>    |                     |                             | command line is recommended to ensure it works. Strings with more than 1000 characters are not allowed. |

Reviewer: "More than x characters" is oddly stated.  Flipping the logic makes it clearer.  (Table space is broken, please test.)
```suggestion
   |                     |                             | command line first is recommended. Strings up to 1000 characters are allowed. |
```
